common/openstack/adding-an-openstack-infrastructure-provider.adoc

```diff
@@ -0,0 +1,57 @@
+
+. In the *Default* tab, under *Endpoints*, configure the host and authentication details of your OpenStack provider:
+.. Enter the *Host Name or IP address(IPv4 or IPv6)* of the provider. If your provider is an undercloud, use its hostname (see https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/single/director-installation-and-usage/#sect-Setting_the_Hostname_for_the_System[Setting the Hostname for the System] in Red Hat OpenStack Platform _Director Installation and Usage_ for more details)
+.. In *API Port*, set the public port used by the OpenStack Keystone service. By default, OpenStack uses port 5000 for this.
+.. Select the appropriate *Security Protocol* used for authenticating with your OpenStack provider.
+.. In the *Username* field, enter the name of an OpenStack user with privileged access (for example, *admin*). Then, provide its corresponding password in the *Password* and *Confirm Password* fields.
+.. Click *Validate* to confirm {product-title} can connect to the OpenStack provider.
+
+. Next, configure how {product-title} should receive events from the OpenStack provider. Click the *Events* tab in the *Endpoints* section to start.
+* To use the Telemetry service of the OpenStack provider, select *Ceilometer*. Before you do so, the provider must first be configured accordingly. See xref:openstack-events-uc[] for details.
+* If you prefer to use the AMQP Messaging bus instead, select *AMQP*. When you do:
+ In *Hostname (or IPv4 or IPv6 address)* (of the *Events* tab, under *Endpoints*), enter the public IP or fully qualified domain name of the AMQP host.
+** In the *API Port*, set the public port used by AMQP. By default, OpenStack uses port 5672 for this.
+** In the *Username* field, enter the name of an OpenStack user with privileged access (for example, *admin*). Then, provide its corresponding password in the *Password* and *Confirm Password* fields.
+** Click *Validate* to confirm the credentials.
+
+. You can also configure SSH access to all hosts managed by the OpenStack infrastructure provider. To do so, click on the *RSA key pair* tab in the *Endpoints* section.
+.. From there, enter the *Username* of an account with privileged access. 
+.. If you selected *SSL* in *Endpoints > Default > Security Protocol* earlier, use the *Browse* button to find and set a private key.
+
+. Click *Add* after configuring the infrastructure provider.
+
+
+[NOTE]
+====
+{product-title} requires that the `adminURL` endpoint for all OpenStack services be on a non-private network.
+Accordingly, assign the adminURL endpoint an IP address of something other than `192.168.x.x`.
+The `adminURL` endpoint must be accessible to the {product-title} Appliance that is responsible for collecting inventory and gathering metrics from the OpenStack environment. Additionally, all the Keystone endpoints must be accessible, otherwise refresh will fail.
```

Comment:
Fixed in: https://github.com/ManageIQ/manageiq_docs/pull/184/commits/87068f77677f6a031edab31278c1f5272f15b5e0